packages/terra-application-links/src/tabs/_ApplicationTabMenuContent.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import List from 'terra-list';
+import styles from './ApplicationTabMenuContent.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The child tabs to be placed in the menu.
+   */
+  children: PropTypes.node,
+  /**
+   * Prop from popup, determines if the menu height is bound by the screen.
```

Comment:
Nit: since the navigation comments are fairly complex and uses several packages, can we specify terra-popup non-familiar consumers?